### PR TITLE
Switch JUnit tests from v3 to v4

### PR DESCRIPTION
**Pull request by aaime**

### CODE REVIEW

**Comment by aaime** [comment]
Cleaned up all xml files formatted by mistake, as per @bradh feedback

File: geowebcache/core/src/test/java/org/geowebcache/grid/BoundingBoxTest.java [hunk 1]
```diff
@@ -41,26 +43,28 @@ public void testBBOXScale() throws Exception {
...
         BoundingBox bb2 = new BoundingBox(5, 5, 20, 20);
 
         BoundingBox intersection = BoundingBox.intersection(bb1, bb2);
-        assertNotNull(intersection);
-        assertEquals(5D, intersection.getWidth());
-        assertEquals(5D, intersection.getHeight());
-        assertTrue(intersection.isSane());
+        Assert.assertNotNull(intersection);
+        Assert.assertEquals(5D, intersection.getWidth(), 0d);
+        Assert.assertEquals(5D, intersection.getHeight(), 0d);
```
Comment (jodygarnett) [inline geowebcache/core/src/test/java/org/geowebcache/grid/BoundingBoxTest.java:54]: Did a double take at `5D` vs `0d`, but it is not case sensitive ... I just have never see `D` before.

File: geowebcache/core/src/test/java/org/geowebcache/grid/GridCalculatorTest.java [hunk 2]
```diff
@@ -4,24 +4,25 @@
...
 
     GridSetBroker gridSetBroker =
             new GridSetBroker(Collections.singletonList(new DefaultGridsets(false, false)));
 
-    @Override
-    protected void setUp() throws Exception {
-        super.setUp();
-    }
+    @Before
+    public void setUp() throws Exception {}
```
Comment (jodygarnett) [inline geowebcache/core/src/test/java/org/geowebcache/grid/GridCalculatorTest.java:23]: Should this just be removed?

**Review by jodygarnett — APPROVED** [review]

File: geowebcache/core/src/test/java/org/geowebcache/grid/GridCalculatorTest.java (hunk 2 above)
Comment (aaime) [reply geowebcache/core/src/test/java/org/geowebcache/grid/GridCalculatorTest.java:23]: Could indeed.

File: geowebcache/core/src/test/java/org/geowebcache/grid/BoundingBoxTest.java (hunk 1 above)
Comment (aaime) [reply geowebcache/core/src/test/java/org/geowebcache/grid/BoundingBoxTest.java:54]: Both are valid, I prefer the "d" form. The capital one was there already I believe.

**Pull request merged**